packages/modelviewer.dev/data/docs.json

```diff
@@ -175,53 +654,92 @@
         "htmlName": "availableAnimations",
-        "description": "12345"
+        "description": "TODO"
       },
       {
         "name": "currentTime",
         "htmlName": "currentTime",
-        "description": "12345"
+        "description": "This property reports the current track time of the currently selected animation. If no animations are available, the value is always 0. This property can be set in order to seek along the timeline of the currently playing animation. For example, if you set it to 0, it will reset an animation to the beginning."
+      },
+      {
+        "name": "paused",
+        "htmlName": "paused",
+        "description": "This property is read-only. It returns true if animations are paused. It returns false if animations are playing. Animations always start paused, and remain so unless the autoplay attribute is set or the .play() method is invoked."
       }
     ],
     "Methods": [
       {
         "name": "play()",
         "htmlName": "play",
-        "description": "12345"
+        "description": "Causes animations to be played. Use the autoplay attribute if you want animations to be played automatically. If there are no animations, nothing will happen, so make sure that the model is loaded before invoking this method."
       },
       {
         "name": "pause()",
         "htmlName": "pause",
-        "description": "12345"
+        "description": "Causes animations to be paused. If you want to reset the current animation to the beginning, you should also set the currentTime property to 0."
       }
     ],
     "Events": [
       {
         "name": "play",
         "htmlName": "play",
-        "description": "12345"
+        "description": "Dispatched when animations begin to play."
       },
       {
         "name": "pause",
         "htmlName": "pause",
-        "description": "12345"
+        "description": "Dispatched when animations are paused. A model always begins in the paused state, so it is worth mentioning that this event will not be dispatched until the the pause() method is invoked after animations have begun playing."
+      }
+    ]
+  },
+  {
+    "Title": "Scene Graph",
+    "Properties": [
+      {
+        "name": "model",
+        "htmlName": "model",
+        "description": "TODO."
+      }
+    ],
+    "Methods": [
+      {
+        "name": "exportScene(options)",
+        "htmlName": "exportScene",
+        "description": "Exports the model in glTF or GLB formats. Returns a promise that resolves in a Blob of type \"application/octet-stream\" or \"application/json\" depending on the export format. For a list of parameters see <a href='https://threejs.org/docs/index.html#examples/en/exporters/GLTFExporter.parse'> GLTF exporter</a> documentation, with the exception that all animations are automatically exported and truncateDrawRange defaults to true."
+      }
+    ],
+    "Events": [
+      {
+        "name": "scene-graph-ready",
+        "htmlName": "sceneGraphReady",
```

Comment:
Is the camel case intentional here?

Reply:
in the htmlName or sceneGraphReady?

Reply:
sceneGraphReady. It's an event, so I didn't think it had a camel-case version. Am I wrong?

Reply:
that's just for giving it an id and url location that works in my code

Reply:
oh, I see. I misunderstood because the attributes have a kebab-case version and a camel-case version (for JS), which I thought was what htmlName meant. Disregard.

Reply:
👍